content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -104,18 +96,20 @@ istio-pilot-58c65f74bc-2f5xn             1/1       Running   0          1m
 
 * For option 2, uninstall using Helm:
 
+> Uninstalling this chart does not delete Istio's registered CRDs. Istio, by design, expects
+> CRDs to leak into the Kubernetes environment. As CRDs contain all runtime configuration
+> data in custom resources the Istio designers consider better to explicitly delete this
+> configuration rather then unexpectedly to lose it.
+
     {{< text bash >}}
     $ helm delete --purge istio-minimal
+    $ helm delete --purge istio-init
     {{< /text >}}
 
-    If your Helm version is less than 2.10.0, then you need to manually cleanup extra job resource before redeploy new version of Istio chart:
-
-    {{< text bash >}}
-    $ kubectl -n istio-system delete job --all
-    {{< /text >}}
+* If desired, run the following command to delete all CRDs:
 
-* If desired, delete the CRDs:
+> {{< warning_icon >}} Deleting CRDs deletes any configuration changes that you have made to Istio.
 
     {{< text bash >}}
-    $ kubectl delete -f install/kubernetes/helm/istio/templates/crds.yaml
+    $ for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl delete -f $i; done
```

Comment:
Is this the only cleanup step that needs to be done between istio installations?

Reply:
Yes, this is the last step if users want to cleanup all istio `CRDs`. The namespace should be deleted if they choose `kubectl delete...` instead of `helm delete`. But `CRDs` are cluster-wide resources, so it doesn't matter if the namespace exists or not.

Reply:
@GregHanson - depends on the workflow.  For option 1, the namespace needs to be deleted.  For option 2, the namespace is automatically deleted.